From: Jiri Slaby <jslaby@suse.cz>
Subject: kABI: protect struct ccw_device_private
Patch-mainline: never, kabi
References: kabi

In 4.4.134, commit 40b64c59f35505ea9cb250537a95f8915355e87e (s390/cio:
fix return code after missing interrupt), upstream commit
770b55c995d171f026a9efb85e71e3b1ea47b93d added async_kill_io_rc int to
struct ccw_device_private. It made the kABI checker to complain.

Given the structure is internal, just hide the change from the kABI
checker.

Signed-off-by: Jiri Slaby <jslaby@suse.cz>
---
 drivers/s390/cio/io_sch.h |    2 ++
 1 file changed, 2 insertions(+)

--- a/drivers/s390/cio/io_sch.h
+++ b/drivers/s390/cio/io_sch.h
@@ -155,7 +155,9 @@ struct ccw_device_private {
 	unsigned long intparm;	/* user interruption parameter */
 	struct qdio_irq *qdio_data;
 	struct irb irb;		/* device status */
+#ifndef __GENKSYMS__
 	int async_kill_io_rc;
+#endif
 	struct senseid senseid;	/* SenseID info */
 	struct pgid pgid[8];	/* path group IDs per chpid*/
 	struct ccw1 iccws[2];	/* ccws for SNID/SID/SPGID commands */
